### PR TITLE
Update image lightbox styles & settings

### PR DESCRIPTION
**Pull request by sashachabin**
- Update veil styles & buttons 
- Update animations speed
- Add `closeOnPullDown`
- Add zoom in / zoom out cursor
- Remove unused `useState`s, refactor variables

### CODE REVIEW

File: utils/notionTypeParser/imageParser.jsx [hunk 1]
```diff
@@ -3,71 +3,53 @@ import Image from 'next/image'
...
-        doubleClickMaxStops,
-        keyboardMoveDistance,
-        wheelZoomDistanceFactor,
-        pinchZoomDistanceFactor,
-        scrollToZoom,
-    }
-
     const image = (
         <>
+            {/* Add zoom-in / zoom-out cursors */}
```
Comment (AtkishkinVlad) [inline utils/notionTypeParser/imageParser.jsx:29]: **question:** maybe can remove comment ?

File: utils/notionTypeParser/imageParser.jsx [hunk 2]
```diff
@@ -3,71 +3,53 @@ import Image from 'next/image'
...
+    ZOOM_SETTINGS,
+} from '../../consts/lightboxOptions'
 
 function GuideImage({ notionType }) {
     const cn = classNames(styles.Manual__image, {
         [styles.Manual__image_svg]: String(notionType.content.image_name).includes('svg'),
     })
 
     const [open, setOpen] = useState(false)
-    const [animationDuration, setAnimationDuration] = useState(ANIMATION_DURATION)
```
Comment (AtkishkinVlad) [inline utils/notionTypeParser/imageParser.jsx:29]: **praise:** 🔥💖🔥

File: consts/lightboxOptions.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    },
+}
+
+export const CONTROLLER_SETTINGS = {
+    closeOnBackdropClick: true,
+    closeOnPullDown: true,
+}
+
+export const CAROUSEL_SETTINGS = {
+    isFinite: true,
```
Comment (AtkishkinVlad) [inline consts/lightboxOptions.js:22]: **question:** maybe disable the endless carousel and the carousel effect in general ?

File: utils/notionTypeParser/imageParser.jsx (hunk 1 above)
Comment (sashachabin) [reply utils/notionTypeParser/imageParser.jsx:29]: I suggest you save it, it's a hack

**Review by AtkishkinVlad — APPROVED** [review]

**Pull request merged**